gameinfos.inc.php

```diff
@@ -76,7 +76,19 @@
 // Games categories
 //  You can attribute any number of "tags" to your game.
 //  Each tag has a specific ID (ex: 22 for the category "Prototype", 101 for the tag "Science-fiction theme game")
-'tags' => array(3, 12, 20, 102, 106, 200, 207, 208, 209, 210),
+// @see https://en.doc.boardgamearena.com/Game_meta-information:_gameinfos.inc.php#Tags
+'tags' => [
+    3, // for regular players
+    12, // long game >30m
+    20, // awarded game
+    102, // historical
+    106, // building
+    200, // cards
+    207, // combos
+    208, // area majority
+    209, // race
+    210 // collection
```

Comment:
To make future additions less error-prone, what do you think of adding a comma after 210?

We aren't doing this consistently in our PHP files yet, but I'm trying to move us in that direction incrementally.

Reply:
Happy to! Added.